### DISCUSSION TITLE
Reduce data being used because of ngrok

### DISCUSSION THREAD

**Discussion by sitolam** [lllyasviel/Fooocus#2252] [Q&A]
I'm running Fooocus on kaggle, the problem that I have is that the output data limit of ngrok runs out quickly. I would like to reduce the data being used by for example disabling all images shown on the website. Is it possible to disable all the images not only the preview images but also the output images and are there any other ways to reduce the data that is being used by the website? If someone knows how to fix that the `--share` feature gives an error on kaggle, let me know to. Thanks in advance!

**Comment by mashb1t** [comment]
Kaggle issue link: https://github.com/lllyasviel/Fooocus/issues/2257

**Comment by mashb1t** [comment]
You've already found the preview image hiding. I've also created an intermediate image hiding in PR https://github.com/lllyasviel/Fooocus/pull/1013 which you can further modify to not show any images (in webui.py, only yield gr.update() instead of the image in the output instructuons after generation). Hope this helps!
